src/index.less

```diff
@@ -5,7 +5,7 @@
 @maxMobile: ~"all and (max-width: 420px)";
 @minMobile: ~"all and (min-width: 420px)";
 
-@import (less) "~react-day-picker/lib/style.css";
```

Comment:
Dette var noe jeg la inn for å få det til å kjøre lokalt da jeg skulle se på testene, ta gjerne en dobbeltsjekk på på om det blir riktig om du ikke har gjort det allerede!